src/main/java/core/basesyntax/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.report;
+
+import core.basesyntax.storage.FruitShop;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("fruit,quantity\n");
```

Comment:
create constant for `"fruit,quantity"`
```suggestion
        sb.append("fruit,quantity");
```